jetty-home/src/main/resources/modules/hawtio.mod

```diff
@@ -26,6 +26,9 @@ http://hawt.io/
 http://github.com/hawtio/hawtio
 http://www.apache.org/licenses/LICENSE-2.0.html
 
+[ini]
+hawtio.version?=2.0.2
```

Comment:
@janbartel @joakime  maybe last 2.13.5? (I have simply put the first version working correctly with jdk11)

Reply:
change default to last 2.13.5